src/effects/effectchainslot.cpp

```diff
@@ -222,13 +219,6 @@ void EffectChainSlot::loadEffectChain(EffectChainPointer pEffectChain) {
         // not of the chain. Propagate the current settings to the chain.
         m_pEffectChain->setMix(m_pControlChainMix->get());
         m_pEffectChain->setEnabled(m_pControlChainEnabled->get() > 0.0);
-        foreach (ChannelInfo* pChannelInfo, m_channelInfoByName) {
```

Comment:
I think we need to do here something alternative. Else we brake the effect chain layer. This might be usefull for loading predefined effect chains later. Currently used in legacy LateNight only.

Reply:
AFAIK this works as is and I'm hesitant to change it further. The duplicate maintenance of the state of the routing switches between EffectChainSlot and EffectChain makes this really brittle.

Reply:
There is actually an issue. If I load an effect and enable it for deck 1, than go to Latenight, just change the chain, the old effect stops and the new one is disabled, which is unexpected.
If I now go back to Deere, I have no chance to make effects working again. 
This is an unusual use-case, but we should keep effect chain changing working for now.

Reply:
:/ Switching chains should not have been included in the initial implementation of the effects system. Here we suffer the consequences...

Reply:
This turns out to be really complicated because the architecture is so messy. I will fix that bug you mentioned and leave cleaning up the architecture as a TODO for now.

Reply:
Thank you.Part of the issue here, is that this PR not only moved the effects post fader, it also includes some first refactoring steps for an unconsolidated final plan. That makes reviewing quit hard.

Reply:
I can confirm that this is fixed now.